test/index.js

```diff
@@ -383,7 +386,8 @@ test('toString', function() {
   eq(Z.toString(['foo', 'bar']), '["foo", "bar"]');
   eq(Z.toString(/x/.exec('xyz')), '["x", "index": 0, "input": "xyz"]');
   eq(Z.toString((function() { var xs = []; xs.z = true; xs.a = true; return xs; }())), '["a": true, "z": true]');
-  eq(Z.toString(circular), '["foo", <Circular>]');
+  eq(Z.toString(ones), '[1, <Circular>]');
+  eq(Z.toString(ones_), '[1, [1, <Circular>]]');
```

Comment:
Having two string representations of one value is somewhat disturbing, but I don't see an alternative.